docs/syntax.md

```diff
@@ -0,0 +1,340 @@
+- `GN` ![##00ff00](https://via.placeholder.com/15/00ff00/000000?text=+) (green)
+- `TQ` ![##00ffff](https://via.placeholder.com/15/00ffff/000000?text=+) (turquoise)
+- `LB` ![##a0dfff](https://via.placeholder.com/15/a0dfff/000000?text=+) (light blue)
+- `BU` ![##0066ff](https://via.placeholder.com/15/0066ff/000000?text=+) (blue)
+- `VT` ![##8000ff](https://via.placeholder.com/15/8000ff/000000?text=+) (violet)
+- `BN` ![##895956](https://via.placeholder.com/15/895956/000000?text=+) (brown)
+- `BG` ![##ceb673](https://via.placeholder.com/15/ceb673/000000?text=+) (beige)
+- `IV` ![##f5f0d0](https://via.placeholder.com/15/f5f0d0/000000?text=+) (ivory)
+- `SL` ![##708090](https://via.placeholder.com/15/708090/000000?text=+) (slate)
+- `CU` ![##d6775e](https://via.placeholder.com/15/d6775e/000000?text=+) (copper)
+- `SN` ![##aaaaaa](https://via.placeholder.com/15/aaaaaa/000000?text=+) (tin)
+- `SR` ![##84878c](https://via.placeholder.com/15/84878c/000000?text=+) (silver)
+- `GD` ![##ffcf80](https://via.placeholder.com/15/ffcf80/000000?text=+) (gold)
+
+<!-- color list generated with a helper script: -->
+<!-- https://gist.github.com/formatc1702/3c93fb4c5e392364899283f78672b952 -->
+
+## Cable color codes
+
+Supported color codes:
+
+- `DIN` for [DIN 47100](https://en.wikipedia.org/wiki/DIN_47100)
+
+  ![##ffffff](https://via.placeholder.com/15/ffffff/000000?text=+)
+  ![##895956](https://via.placeholder.com/15/895956/000000?text=+)
+  ![##00ff00](https://via.placeholder.com/15/00ff00/000000?text=+)
+  ![##ffff00](https://via.placeholder.com/15/ffff00/000000?text=+)
+  ![##999999](https://via.placeholder.com/15/999999/000000?text=+)
+  ![##ff66cc](https://via.placeholder.com/15/ff66cc/000000?text=+)
+  ![##0066ff](https://via.placeholder.com/15/0066ff/000000?text=+)
+  ![##ff0000](https://via.placeholder.com/15/ff0000/000000?text=+)
+  ![##000000](https://via.placeholder.com/15/000000/000000?text=+)
+  ![##8000ff](https://via.placeholder.com/15/8000ff/000000?text=+)
+  ...
+
+- `IEC` for [IEC 62](https://en.wikipedia.org/wiki/Electronic_color_code#Color_band_system) ("ROY G BIV")
+
+  ![##895956](https://via.placeholder.com/15/895956/000000?text=+)
+  ![##ff0000](https://via.placeholder.com/15/ff0000/000000?text=+)
+  ![##ff8000](https://via.placeholder.com/15/ff8000/000000?text=+)
+  ![##ffff00](https://via.placeholder.com/15/ffff00/000000?text=+)
+  ![##00ff00](https://via.placeholder.com/15/00ff00/000000?text=+)
+  ![##0066ff](https://via.placeholder.com/15/0066ff/000000?text=+)
+  ![##8000ff](https://via.placeholder.com/15/8000ff/000000?text=+)
+  ![##999999](https://via.placeholder.com/15/999999/000000?text=+)
+  ![##ffffff](https://via.placeholder.com/15/ffffff/000000?text=+)
+  ![##000000](https://via.placeholder.com/15/000000/000000?text=+)
+  ...
+
+- `TEL` and `TELALT`  for [25-pair color code](https://en.wikipedia.org/wiki/Electronic_color_code#Color_band_system)
```

Comment:
I assume the correct URL is `https://en.wikipedia.org/wiki/25-pair_color_code`, but double-check with the contributor. Where did you copy the existing link from?

Reply:
You are correct. `README.md` links to the correct Wikipedia page.